clevertap-core/src/main/java/com/clevertap/android/sdk/db/CtDatabase.kt

```diff
@@ -80,11 +82,17 @@ class DatabaseHelper internal constructor(val context: Context, val config: Clev
                 executeStatement(db, CREATE_NOTIFICATION_VIEWED_TABLE)
                 executeStatement(db, NOTIFICATION_VIEWED_INDEX)
                 migrateUserProfilesTable(db)
+                executeStatement(db, CREATE_USER_EVENT_LOGS_TABLE)// when app updates 2 to 5
             }
 
             3 -> {
                 // For DB Version 4, just migrate userProfiles table
                 migrateUserProfilesTable(db)
+                executeStatement(db, CREATE_USER_EVENT_LOGS_TABLE)// when app updates 3 to 5
+            }
+
+            4 -> {
```

Comment:
It would be more maintainable to add the migration in an "if (oldVersion<5)" outside of the "when" and not duplicate the logic for every version.

Reply:
fixed in https://github.com/CleverTap/clevertap-android-sdk/commit/f7a8497ccf9a3acf0e62079e04bcf2114c1fc8bf